packages/ember-glimmer/lib/utils/bindings.js

```diff
@@ -14,6 +14,17 @@ function referenceForKey(component, key) {
 }
 
 function referenceForParts(component, parts) {
+  let isAttrs = parts[0] === 'attrs';
```

Comment:
is parts guaranteed to be length 1? is it very important that we never index outside of the array on get.